src/store.js

```diff
@@ -0,0 +1,15 @@
+import { createStore, applyMiddleware } from 'redux';
+import thunk from 'redux-thunk';
+import { createLogger } from 'redux-logger';
+
+const middlewares = [thunk];
+
+if (process.env.NODE_ENV === 'development') {
+    const logger = createLogger();
```

Comment:
To improve the output we should initialize the logger using `createLogger({ diff: true, collapsed: true })`; this will collapse the actions by default and also output a diff of state changes.

Reply:
Thanks. Will do.